pkg/ebpf/tracee.go

```diff
@@ -1731,6 +1731,14 @@ func (t *Tracee) invokeInitEvents(out chan *trace.Event) {
 
 	// Initial namespace events
 
+	matchedPolicies = policiesMatch(t.eventsState[events.TraceeInfo])
+	if matchedPolicies > 0 {
```

Comment:
I'm getting rid of this `eventState` direct access soon. JFIY. Lookups related to applied events will be done via t.policyManager.